src/Account.sol

```diff
@@ -197,6 +152,13 @@ contract Account is IAccount, OpsReady, Auth, Initializable {
                                OWNERSHIP
     //////////////////////////////////////////////////////////////*/
 
+    /// @inheritdoc IAccount
+    function setInitialOwnership(address _owner) external override {
+        if (msg.sender != address(FACTORY)) revert Unauthorized();
+        owner = _owner;
+        emit OwnershipTransferred(address(0), _owner);
+    }
```

Comment:
**Praise/Thought:** I like this solution a lot. Very simple and it removes a bunch of code 🙌 🎉. One thing I am wondering is if we should ensure with a flag that it is only ever called once? Seems like it shouldn't be a problem, as it can only be called once anyway in the `newAccount` function.